PSKNodeServer/PSKNodeServer/API/scanditScan/ScanditScannerViewController.swift

```diff
@@ -6,22 +6,78 @@
 //
 
 import UIKit
+import ScanditBarcodeCapture
 
 class ScanditScannerViewController: UIViewController {
     @IBOutlet weak var apiKeyLabel: UILabel!
     private let scanditApiKey: String
+    private let symbologies: [Symbology]
+    private let dataCaptureContext: DataCaptureContext
+    private var barcodeCapture: BarcodeCapture
     
     override func viewDidLoad() {
         super.viewDidLoad()
         apiKeyLabel.text = scanditApiKey
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        barcodeCapture.addListener(self)
+        
+        let cameraSettings = BarcodeCapture.recommendedCameraSettings
+        Camera.default?.apply(cameraSettings)
+        dataCaptureContext.setFrameSource(Camera.default)
+        Camera.default?.switch(toDesiredState: .on)
+        
+        let captureView = DataCaptureView(context: dataCaptureContext, frame: view.bounds)
+        captureView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(captureView)
+        
+        _ = BarcodeCaptureOverlay(barcodeCapture: barcodeCapture, view: captureView)
+    }
+    
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    init(with scanditApiKey: String){
+    init(with scanditApiKey: String, andWith symbologies: [Symbology]){
         self.scanditApiKey = scanditApiKey
+        self.symbologies = symbologies
```

Comment:
Do we want to have some check whether array is not empty? I'm not sure what will happen with scanning process when no symbology will be set (infinite scanning?)

Reply:
As discussed - the scanner will recognize only given symbologies. But if composite scan is enabled, it still works even if symbologies are empty. Considering this is a programatically given constant, I'd consider adding integration test instead, but otherwise this is something you find during development, there is no sudden runtime error possible because user can't change this.